pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
Since we don't have multiple flavors of GDS unit tests, are markers necessary?

Reply:
Ok, I'll remove the markers from the tests